diff --git a/src/serialization/resources/empathicVoice/resources/chat/index.ts b/src/serialization/resources/empathicVoice/resources/chat/index.ts
index f5ca113c..d9adb1af 100644
--- a/src/serialization/resources/empathicVoice/resources/chat/index.ts
+++ b/src/serialization/resources/empathicVoice/resources/chat/index.ts
@@ -1,7 +1,2 @@
 export * from "./client/index.js";
 export * from "./types/index.js";
-/**
- * @deprecated Use `serialization.empathicVoice.SubscribeEvent` instead.
- * This serializer alias will be removed in a future version.
- */
-export { SubscribeEvent } from "./types/SubscribeEvent.js";
